tck/src/main/java/org/eclipse/microprofile/config/tck/util/AdditionalAssertions.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.microprofile.config.tck.util;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.IntStream;
+
+import org.testng.Assert;
+
+/**
+ *
+ * @author Joseph Cass
+ */
+public final class AdditionalAssertions {
+
+    private AdditionalAssertions() {
+        // utility class
+    }
+
+    public static void assertURLArrayEquals(URL[] value, URL[] expectedValue) throws MalformedURLException {
+        assertURLListEquals(Arrays.asList(value), Arrays.asList(expectedValue));
+    }
+
+    public static void assertURLListEquals(List<URL> value, List<URL> expectedValue) throws MalformedURLException {
```

Comment:
What do you think if we repleace `List` to `Iterable`

Reply:
The rest of the class only does Array, List and Set comparisons, where the order of the List comparisons matters, but the order of the set comparisons doesn't (despite being a `LinkedHashSet`) so I'm not sure if this is worth it.